nimare/tests/new_json_format/dset_study1.json

```diff
@@ -0,0 +1,75 @@
+{
+  "Studies": [
+    {
+      "Groups": [
+        {
+          "@id": "gg_id1",
+          "Name": "group1",
+          "N": 12
+        }
+      ],
+      "Conditions": [
+        {
+          "@id": "cc_id1",
+          "Name": "household items"
+        },
+        {
+          "@id": "cc_id2",
+          "Name": "animals"
+        }
+      ],
+      "@id": 9593960,
+      "Analyses": [
+        {
```

Comment:
Each `Analysis` should probably have a mandatory `"Label"` or `"Name"` property, and a strongly encouraged `"Description"` property.

Reply:
Agreed